web/pdf_sidebar.js

```diff
@@ -104,7 +106,7 @@ var PDFSidebar = (function PDFSidebarClosure() {
     reset: function PDFSidebar_reset() {
       this.isInitialViewSet = false;
 
-      this.close();
+      this._hideUINotification(null);
```

Comment:
Is it intended that the `close()` call is removed here? If so, what is the rationale for that?